jmh-core/src/test/java/org/openjdk/jmh/util/TestListStatistics.java

```diff
@@ -187,7 +187,7 @@ public strictfp void testSignificant_Always() {
 
         for (double conf : new double[] {0.5, 0.9, 0.99, 0.999, 0.9999, 0.99999}) {
             Assert.assertTrue("Diff significant at " + conf, s1.isDifferent(s2, conf));
-            Assert.assertEquals(1, s1.compareTo(s2, conf));
+            Assert.assertEquals(-1, s1.compareTo(s2, conf));
```

Comment:
Technically speaking, the assert should be `less than 0` rather than `equal to -1`.
`-2` is a valid result of `compareTo` call.

WDYT?

Reply:
You need to press the checkbox in the comment to let me see the comment. But I guess what it says...

Reply:
Yes, true. Fixed in the new commit.